app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
         @visible_rentals = current_account.rentals.visible
         @remote_accounts = current_account.remote_accounts
         @remote_rentals_by_account = current_account.remote_rentals.ordered
-          .includes(:remote_account).group_by(&:remote_account)
```

Comment:
looks weird to make model responsible for knowing what includes are needed in the controller
maybe get a private method `remote_rentals`
that could be overridden in the apps?